src/bundle/Resources/views/admin/search/search_form.html.twig

```diff
@@ -10,6 +10,100 @@
+    </div>
+    <div class="ez-filters__btns">
+        <button class="btn btn-dark ez-btn-clear">
+            {{ 'search.clear'|trans|desc('Clear') }}</button>
+        <button type="submit" class="btn btn-secondary ez-btn-apply" disabled>
+            {{ 'search.applay'|trans|desc('Apply') }}</button>
+    </div>
+</div>
+{{ form_widget(form.last_modified, {'attr': {'hidden': 'hidden'}}) }}
+{{ form_widget(form.created, {'attr': {'hidden': 'hidden'}}) }}
+<div class="modal fade ez-modal ez-modal--select-modified-range" tabindex="-1" role="dialog" data-period-selector="#search_last_modified_date_interval" data-end-selector="#search_last_modified_end_date">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">{{ 'search.date.range'|trans|desc('Select date range') }}</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <label class="ez-filters__item-label">{{ 'search.date.from'|trans|desc('From:') }}
+                    <input type="text" class="form-control ez-date-select ez-date-select--start" data-target-selector="#modified-start-range">
+                    <input type="text" id="modified-start-range" hidden>
+                </label>
+                <label class="ez-filters__item-label">{{ 'search.date.to'|trans|desc('To:') }}
+                    <input type="text" class="form-control ez-date-select ez-date-select--end" data-target-selector="#modified-end-range">
+                    <input type="text" id="modified-end-range" hidden>
+                </label>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-dark" data-dismiss="modal">
+                    {{ 'modal.cancel'|trans|desc('Cancel') }}
+                </button>
+                <button class="btn btn-secondary ez-btn--select" data-dismiss="modal" disabled>
+                    {{ 'modal.select'|trans|desc('Select') }}
+                </button>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="modal fade ez-modal ez-modal--select-created-range" tabindex="-1" role="dialog" data-period-selector="#search_created_date_interval" data-end-selector="#search_created_end_date">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">{{ 'search.date.range'|trans|desc('Select date range') }}</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
```

Comment:
Please, use the SVG icon with `discard` identifier instead.

Reply:
This would be the first modal with SVG `discard` icon, we should be consistent in the whole admin.